pfrl/agents/goal_conditioned_td3.py

```diff
@@ -100,8 +107,15 @@ def __init__(
         self.buffer_freq = buffer_freq
         self.minibatch_size = minibatch_size
         self.add_entropy = add_entropy
+
         if add_entropy:
             self.temperature = 1.0
+        self.q_func1_variance_record = collections.deque(maxlen=10)
+        self.q_func2_variance_record = collections.deque(maxlen=10)
```

Comment:
why 10 of them? cuz of 10 steps?

Reply:
just an arbitary value -- I'll make it in argument

Reply:
fixed